packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -118,7 +128,10 @@ export class BlockList extends Component {
 					innerRef={ this.scrollViewInnerRef }
 					extraScrollHeight={ innerToolbarHeight + 10 }
 					keyboardShouldPersistTaps="always"
-					scrollViewStyle={ { flex: isRootList ? 1 : 0 } }
+					scrollViewStyle={ {
+						flex: isRootList ? 1 : 0,
+						overflow: 'visible',
```

Comment:
as far as I remember from the conversation with Piotr the `overflow: 'visible'` is to prevent edge to be cut is some cases. In example without it in Column PR borders was cropped.

However I didn't tested the latest version and if you do not noticed any difference by removing this I will try to confirm that on Android as well and let you know.

If not the possible solution is to remove `overflow: 'visible'`  only for root list - we already have a prop called `isRootList`

Reply:
Targeting current master branch it is ok to remove that prop. I double check and borders for currently available blocks seems to work as expected without it. 

Will test in Columns PR as well later. 

However we discussed with Piotr that we can remove it from this PR and if necessary add in [Column PR adjustment](https://github.com/WordPress/gutenberg/pull/21073). That will make more sense because according to above it is adjustments for Columns

Reply:
I removed it since it is not needed at this stage. I think if it will be needed for columns, we should add it in columns PR.